js/script.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const levels = document.querySelector('.levels');
+let levelTable = document.querySelectorAll('.level');
+let tableNumber = 0;
+levelTable = [].slice.call(levelTable);
+
+levels.addEventListener('click', (e) => {
+  if (e.target && e.target.classList.contains('level')) {
+    tableNumber = levelTable.indexOf(event.target);
+    levelTable[tableNumber].classList.add('selected-level', 'lvl-active');
+    
+    levelTable.forEach((item, i) => {
+      if (i !== tableNumber) {
+        item.classList.remove('selected-level', 'lvl-active');
+      }
+    });
+  }
+});
+
+const start = document.querySelector('.play');
+start.addEventListener('click', render);
+
+function render() {  
+  let cardsTotal;
+  let content = `
+  <div class="card-wrapper">
+    <div class="card-inner">
+      <div class="front-side"></div>
+      <div class="back-side"></div>
+    </div>
+  </div>`;
+  const parent = document.querySelector('.game');
+  let activeTable = document.createElement('div');
+  
+{/* <img class="result" src="./img/game-over.png" alt="лицевая сторона карты"></img> */}
+{/* <img class="card" src="./img/back-side.png" alt="обратная сторона карты"></img> */}
+
+  activeTable.classList.add('table', `table-${tableNumber}`);
+
+  if (tableNumber === 0) {
+    cardsTotal = 3;
+  } else if (tableNumber === 1) {
+    cardsTotal = 6;
+  } else {
+    cardsTotal = 9;
+  } 
+  
+  (() => {
+    for (let i = 0; i < cardsTotal; i++) {
+      activeTable.innerHTML += content;
```

Comment:
Codacy found an issue: [Unsafe assignment to innerHTML](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)